packs/feat-effects/effect-azarketi-purification.json

```diff
@@ -25,7 +25,7 @@
                 "key": "FlatModifier",
                 "label": "Azarketi Purification (vs. ongoing saves against poisons already in your body)",
                 "predicate": [
-                    "poison"
+                    "item:trait:poison"
```

Comment:
Probably need another term in the predicate like `ongoing-poison`, as it doesn't grant the bonus against new poisons.

Reply:
And the custom label may not be needed?